### PR TITLE
RWA-2775- Fix unit test failure WA-case-event-handler

### PR DESCRIPTION
**Pull request by skandala-cgi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-2775

### Change description ###

Fix unit test failure WA-case-event-handler

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/handlers/ReconfigurationCaseEventHandlerTest.java [hunk 1]
```diff
@@ -507,4 +497,17 @@ private EvaluateDmnRequest buildEvaluateUpdateDmnRequest() {
...
 
+    private void assertConsoleOutputHasMessages(CapturedOutput output) {
+        await().ignoreException(Exception.class)
+            .pollInterval(100, MILLISECONDS)
+            .atMost(5, SECONDS)
+            .untilAsserted(() -> {
+                Assertions.assertTrue(output.getOut().contains(RECONFIGURATION_EVENT_INFORMATION_LOG));
+                Assertions.assertTrue(output.getOut().contains(SEND_RECONFIGURATION_REQUEST_LOG));
+                Assertions.assertTrue(output.getOut().contains(RECONFIGURATION_COMPLETED_LOG));
+            });
```
Comment (raghera) [inline src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/handlers/ReconfigurationCaseEventHandlerTest.java:508]: This looks a lot better but are there any other tests which would benefit from the same improvement?  Would be good to catch them all at once.

Comment (raghera) [reply src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/handlers/ReconfigurationCaseEventHandlerTest.java:508]: 👍

**Review by raghera — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/CleanUpMessageJobTest.java [hunk 2]
```diff
@@ -115,10 +116,20 @@ void should_clean_messages_when_environment_is_non_prod(CapturedOutput output) {
...
+    }
 
-        assertTrue(output.getOut().contains(expectedLogMessage));
-        assertTrue(output.getOut().contains("CLEAN_UP_MESSAGES job completed"));
+    private void assertConsoleOutputHasMessages(CapturedOutput output, String expectedLogMessage) {
+        await().ignoreException(Exception.class)
+            .pollInterval(100, MILLISECONDS)
+            .atMost(5, SECONDS)
+            .untilAsserted(() -> {
+                assertTrue(output.getOut().contains(expectedLogMessage));
```
Comment (BenNzewi) [inline src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/CleanUpMessageJobTest.java:127]: Looks like a good time to also make the assertions consistent using Assertions.assert?? (Junit5)

Comment (skandala-cgi) [reply src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/CleanUpMessageJobTest.java]: updated

**Review by BenNzewi — APPROVED** [review]

**Review by raghera — APPROVED** [review]

**Pull request merged**